ingest/sqs/pools/transformer/pool_transformer_test.go

```diff
@@ -491,6 +492,7 @@ func (s *PoolTransformerTestSuite) TestProcessBlock() {
 	s.Require().Equal(poolsData.CosmWasmPoolID, allPools[4].GetId())
 
 	// Validate taker fee for the custom pool
+	// TODO: GetTakerFee in the sqsdomain repo needs to be updated to no longer sort lexicographically
```

Comment:
Will create an issue before we merge but after it is approved.

Reply:
Flagging this so we don't forget

Reply:
https://github.com/osmosis-labs/sqs/issues/289